openshift/patches/004-grpc.patch

```diff
@@ -20,20 +20,20 @@ index 6730081d1..c0709521f 100644
  	if !hasPort(host) {
  		host = net.JoinHostPort(host, defaultPort)
  	}
-@@ -82,12 +78,6 @@ func dial(ctx *TestContext, host, domain string) (*grpc.ClientConn, error) {
+@@ -83,12 +79,6 @@ func dial(ctx *TestContext, host, domain string) (*grpc.ClientConn, error) {
  	}
  
- 	secureOpt := grpc.WithInsecure()
+ 	creds := insecure.NewCredentials()
```

Comment:
Is this the cause? What was failing?

Reply:
It is just a patch confliction as upstream changed the code. We can find the log by following the Slack notification https://coreos.slack.com/archives/CF5ANN61F/p1642552480024100

Here is the logs:
```
09:34:40  error: patch failed: test/e2e/grpc_test.go:34
09:34:40  error: test/e2e/grpc_test.go: patch does not apply
```

We need the gRPC patch as OpenShift Route does not support gRPC.